Core/src/org/sleuthkit/autopsy/timeline/TimeLineController.java

```diff
@@ -116,7 +118,8 @@
 
     private static final ReadOnlyObjectWrapper<TimeZone> timeZone = new ReadOnlyObjectWrapper<>(TimeZone.getDefault());
 
```

Comment:
Even though netbeans compain, we have the convention of using lowercase for logger, since it is not a 'simple' constant.  Similarly timeZone will never refer to another object, but it's wrapped value changes.